### PR TITLE
Replaced image with quote in "Post Link to your repository in chat"

### PR DESCRIPTION
**Pull request by lorugant**
rawgit link:  https://rawgit.com/lorugant/lorugant.github.io/lBrancH/#pages/vi/vi-github-and-markdown.md

A much cleaner version of #1343

### CODE REVIEW

File: pages/vi/vi-github-and-markdown.md [hunk 1]
```diff
@@ -66,7 +66,8 @@ After you fork the repository, you will be on your page: &lt;username&gt;/open-l
 
 After you rename it, make sure to post a link to the github.io website you've just created (&lt;username&gt;.github.io) in the [gitter.im chat](https://gitter.im/open-learning-exchange/chat).
 
-![Example of github.io page](images/vi-example-githubio.png)
+> @dogi  please look at
+https://YourUserName.github.io
```
Comment (xyb994) [inline pages/vi/vi-github-and-markdown.md]: @lorugant Would it look simpler to have the text and url on the same line?

**Comment by lorugant** [comment]
Changed to the same line please check :  5e4ce57

**Comment by xyb994** [comment]
Since this is the [only place](https://github.com/open-learning-exchange/open-learning-exchange.github.io/search?utf8=✓&q=vi-example-githubio.png&type=) using the [screenshot](https://github.com/open-learning-exchange/open-learning-exchange.github.io/blob/master/pages/vi/images/vi-example-githubio.png), should we have the image removed as well?

**Comment by kylemathias** [comment]
yup, remove the image imo

**Comment by xyb994** [comment]
new PR https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/1415

**Pull request closed without merging**